src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+import duke.DukeException;
+import duke.task.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Abstract class to represent the framework of a Command.
+*/
+public abstract class Command {
+
+    protected CommandType commandType;
+    protected boolean active;
```

Comment:
```suggestion
    protected boolean isActive;
```
Perhaps a more intuitive name here? For example isActive